### PR TITLE
Update libraries/joomla/filter/input.php

### PR DESCRIPTION
**Pull request by ced1870**
http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_id=8103&tracker_item_id=29961

### CODE REVIEW

File: libraries/joomla/filter/input.php [hunk 1]
```diff
@@ -259,6 +259,10 @@ public function clean($source, $type = 'string')
 			case 'USERNAME':
 				$result = (string) preg_replace('/[\x00-\x1F\x7F<>"\'%&]/', '', $source);
 				break;
+
```
Comment (elinw) [inline libraries/joomla/filter/input.php:262]: The pull tester says you have some blank space at the end of this line.  Can you update please?
http://developer.joomla.org/pulls/pulls/1805.html

**Comment by dongilbert** [comment]
Remove the trailing whitespace and then update, and we'll be able to get this merged.

**Comment by dongilbert** [comment]
After it's fixed, please leave a comment simply saying "reopen" and it will reopen for you. :+1:

**Comment by ced1870** [comment]
The problem is that I'm not a github master user and I dont know how to remove it and update this !
I find this really diffcult to submit just few lines of codes to improve Joomla! , don't cry then if there is less people helping ...
I have already spent hours to try using github for Joomla but this is not easy for a beginner and I think that you should be able to remove this space too ? 
Regards
Ced

**Comment by dongilbert** [comment]
We can, no problem, but we like it when people who discover bugs / add features get to have their name added to the contributors list. So, we'll get it wrapped up, thanks for finding the issue.

**Comment by ced1870** [comment]
Ok so please remove the space for me :) and add the pull request to joomla. I have to learn how to use github for the next time ! ;)
Thanks !
CEd

**Comment by AmyStephen** [comment]
Very nice of you @dongilbert - @ced1870 - it gets easier each time. Thanks for your patch!

**Comment by nicksavov** [comment]
Hi all,

What's the status on this one?  Doesn't look like it's in staging or master:
https://github.com/joomla/joomla-platform/blob/staging/libraries/joomla/filter/input.php#L261
https://github.com/joomla/joomla-platform/blob/master/libraries/joomla/filter/input.php#L261

**Comment by eddieajau** [comment]
I'm going to merge this because we have other codestyle issues in the Platform currently anyway.

**Comment by nicksavov** [comment]
Great, thanks Andrew et. al.!

**Comment by ienev** [comment]
I know 2.5.x is another branch but are there any plans to fix this in 2.5.x as well? If yes - should I create an entry in the Joomla! Tracker and make a PR?

**Pull request merged**